recipes/in_place/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "in_place" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3d89b9865226410b19fb226f6cf1e1f84976b6347172e390e88f1c96cd731925
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Comment:
Please use the current recommendation for the `script` line from the `example/meta.yaml`:

```
script: "{{ PYTHON }} -m pip install . -vv"
```

Reply:
@synapticarbors sorry I missed that recommendation. It is now updated.